files/es/web/api/domerror/index.md

```diff
@@ -38,14 +38,14 @@ La interfaz **`DOMError`** describe un objeto de error que contiene un nombre de
 | `InvalidNodeTypeError`       | El nodo (o uno de sus antecedentes) es incorrecto.                      |
 | `DataCloneError`             | El objeto no puede clonarse.                                            |
 
-## Compatibilidad de Navegadores
+## Especificaciones
 
-{{Compat("api.DOMError")}}
+{{Specifications}}
 
-## Especificaciones
+## Compatibilidad con navegadores
 
-- [DOM 4 DOMError specification](https://www.w3.org/TR/dom/#interface-domerror)
+{{Compat}}
 
-## Véase también
+## Vér también
```

Comment:
According to `es` conventions

```suggestion
## Véase también
```

Reply:
I've mostly seen "Vér" in place of "Véase" on pages, perhaps we should open an issue regarding that?

Reply:
I don't think an issue would be necessary, you see mostly "Vér" due to there is a lot of content outdated yet, that would be changed eventually